yarn.lock

```diff
@@ -96,11 +96,12 @@
     apollo-datasource-rest "^0.2.0"
     lodash "^4.17.11"
 
-"@apollosproject/data-connector-church-online@^0.8.4":
-  version "0.8.4"
-  resolved "https://registry.yarnpkg.com/@apollosproject/data-connector-church-online/-/data-connector-church-online-0.8.4.tgz#bf4584c89b952d66631cddf6efafb0e1198f1921"
+"@apollosproject/data-connector-church-online@^0.8.7-alpha.1":
+  version "0.8.7-alpha.1"
+  resolved "https://registry.yarnpkg.com/@apollosproject/data-connector-church-online/-/data-connector-church-online-0.8.7-alpha.1.tgz#63ee0a7c0cbc3f1be2a0eca6abe202fabaf7eb60"
+  integrity sha512-SgiKekX+26yXnQ7EIyvy6Fau2NwWEFrvaWf+X1ulomRmdRxdvAWeBUpA4Qv9QiQJT/YIqvzTpKJohLodPgAwoQ==
   dependencies:
-    "@apollosproject/data-schema" "^0.8.4"
+    "@apollosproject/data-schema" "^0.8.5"
```

Comment:
Will this cause us issue since we are still using 0.8.4 in app?

Reply:
@IsaacHardy Vincent says we're "totally fine". 👍